packages/react/src/components/dateInput/components/datePicker/DatePicker.module.scss

```diff
@@ -27,9 +27,12 @@
   box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
   display: inline-block;
   width: 100%;
-  max-width: calc(var(--table-width) + 2 * var(--horizontal-spacing));
-  min-width: var(--container-width-xs);
+  min-width: calc(100vw - 2 * var(--spacing-s));
   padding: var(--vertical-spacing) var(--horizontal-spacing);
+
+  @media only screen and (min-width: 576px) {
```

Comment:
Wonder if we could/should apply the breakpoint from design-tokens: https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/design-tokens/tokens/breakpoint/breakpoint.json ?